spanner/tests/system/test_system.py

```diff
@@ -394,6 +431,94 @@ def _unit_of_work(transaction, name):
                 COUNTERS_TABLE, COUNTERS_COLUMNS, self.ALL))
         self.assertEqual(len(rows), 2)
 
+    def test_multi_processing_map_transaction(self):
+        from multiprocessing import Pool
+        client = Client()
```

Comment:
Why are you creating a client before the fork()?

This is known to cause issues with gRPC, and isn't something we support yet.  If it worked at some point it was purely coincidental.

Reply:
That is what is interesting.  It was working fine before.  But I guess if it is meant to not be supported then I guess this isn't a valid behavior test.